From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jiri Slaby <jslaby@suse.cz>
Date: Mon, 11 Jun 2018 10:44:00 +0200
Subject: [PATCH] fix build with Qt 5.11

The new Qt removed some implicit inclusions of headers. To avoid build
errors, add explicit includes of those we use in the sources.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>

(cherry picked from commit 6c392e903e162b9283e88f53006e929663f2e883)
Signed-off-by: Christian Hesse <mail@eworm.de>
---
 librecad/src/ui/forms/qg_commandwidget.cpp | 2 ++
 librecad/src/ui/generic/widgetcreator.cpp  | 2 ++
 2 files changed, 4 insertions(+)

diff --git a/librecad/src/ui/forms/qg_commandwidget.cpp b/librecad/src/ui/forms/qg_commandwidget.cpp
index 7cd6939..2b90d67 100644
--- a/librecad/src/ui/forms/qg_commandwidget.cpp
+++ b/librecad/src/ui/forms/qg_commandwidget.cpp
@@ -24,6 +24,8 @@
 **
 **********************************************************************/
 #include "qg_commandwidget.h"
+
+#include <QAction>
 #include <QKeyEvent>
 #include <algorithm>
 
diff --git a/librecad/src/ui/generic/widgetcreator.cpp b/librecad/src/ui/generic/widgetcreator.cpp
index ca89ab1..260d9af 100644
--- a/librecad/src/ui/generic/widgetcreator.cpp
+++ b/librecad/src/ui/generic/widgetcreator.cpp
@@ -27,6 +27,8 @@
 #include "widgetcreator.h"
 #include "ui_widgetcreator.h"
 
+#include <QAction>
+#include <QActionGroup>
 #include <QSettings>
 #include <QLineEdit>
 #include <QPushButton>
